tools-local/tasks/mobile.tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -236,6 +237,14 @@ public class ApkBuilder
 
         return (alignedApk, packageId);
     }
+
+    private static string GetRid(string abi) => abi switch 
+        {
+            "arm64-v8a" => "android-arm64",
+            "armeabi-v7a" => "android-arm",
+            "x86_64" => "android-x64",
+            _ => "android-" + abi
+        };
```

Comment:
braces are misaligned :)

Reply:
@akoeplinger will have to do the same for ios and will fix it